### PR TITLE
remove /rest path prefix

### PR DESCRIPTION
**Pull request by hjacobs**
Remove "/rest" prefix, but keep path mapping for compatibility.

### CODE REVIEW

File: src/test/java/de/zalando/zmon/dataservice/proxies/entities/EntitiesControllerTest.java [hunk 1]
```diff
@@ -74,7 +74,7 @@ public void getEntities() throws Exception {
 
     @Test
     public void getEntitiesRest() throws Exception {
-        this.mockMvc.perform(MockMvcRequestBuilders.get("/rest/api/v1/entities?query=htg").header("Authorization", "Bearer 1"))
```
Comment (mohabusama) [inline src/test/java/de/zalando/zmon/dataservice/proxies/entities/EntitiesControllerTest.java]: Shouldn't we keep legacy path test to ensure compatibility? (until it is removed)

**Comment by Jan-M** [comment]
:+1:

File: src/test/java/de/zalando/zmon/dataservice/proxies/entities/EntitiesControllerTest.java (hunk 1 above)
Comment (hjacobs) [inline src/test/java/de/zalando/zmon/dataservice/proxies/entities/EntitiesControllerTest.java]: this one fixes the remaining usage: https://github.com/zalando-zmon/zmon-scheduler/pull/34

**Comment by hjacobs** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**